app/controllers/manifest_uploads_controller.rb

```diff
@@ -1,5 +1,46 @@
 class ManifestUploadsController < ApplicationController
   def new
+  end
+
+  def create
+    manifest = find_or_initialize_manifest
+
+    if manifest.save
+      flash[:notice] = "Upload for manifest #{manifest.tracking_number} submitted successfully."
+      redirect_to root_path
+    end
+  end
+
+  private
+
+  def find_or_initialize_manifest
+    if params[:manifest_id]
+      manifest = Manifest.find(params[:manifest_id])
+      manifest.content[:image] = image_details
```

Comment:
do we only ever support one image per manifest?

`image` is a little non-descriptive for a top-level attribute in the `content`. Could we maybe call it `uploaded_file` or something else?